.github/CODEOWNERS

```diff
@@ -23,11 +23,13 @@
 # The users below will be the default owners for everything in the repository.
 # Unless a line further in this file matches, all of the users here will be
 # requested for review when someone opens a pull request.
-*	@davetcoleman @v4hn @rhaschke 
+*	@davetcoleman @v4hn @rhaschke
 
 # The lines below correspond to directories with a package.xml file and the
 # maintainers specified in that file
```

Comment:
This comment does not correspond with your new line 31, as that's just a file, not a directly. Can we move the files section to the bottom of this? Or keep it at the top but have another comment explaining this?

Reply:
Done